modules/ztp-enabling-assisted-installer-service-on-bare-metal.adoc

```diff
@@ -31,7 +31,7 @@ $ oc edit AgentServiceConfig
 [source,yaml,subs="attributes+"]
 ----
 - cpuArchitecture: x86_64
-    openshiftVersion: "{product-version}"
+    openshiftVersion: "{product-version}.z"
```

Comment:
Not sure we need it in the example. As usually x.y is used, so it could be confusing if illustrated here. I.e. just to be clear that it's optional to use it.